pages/common/pageres.md

```diff
@@ -0,0 +1,29 @@
+# pageres
+
+> Capture screenshots of websites in various resolutions.
+> More information: <https://github.com/sindresorhus/pageres-cli>.
+> More information on API: <https://github.com/sindresorhus/pageres>.
+
+- Basic multi-url, multi-resolution usage:
+
+`pageres {{any_web_page_url}} {{any_web_page_url}} {{1366x768}} {{1600x900}}`
+
+- Override outer option within group:
```

Comment:
I'm not really clear on what this does. Could you perhaps explain?

Reply:
@sbrl 
`pageres [{{https://example.com/}} {{1366x768 1600x900}} --no-crop] [{{https://example2.com/}} {{1024x768}} {{480x320}}] --crop`

Here in the above example `--crop` is for over all group of pages/URL. `--no-crop` is specific for that particular pages/URL, where `--no-crop` overrides `--crop`